viennacl/src/main/scala/org/apache/mahout/viennacl/opencl/javacpp/DenseColumnMatrix.scala

```diff
@@ -36,12 +36,24 @@ import org.bytedeco.javacpp.annotation._
 @Name(Array("viennacl::matrix<double,viennacl::column_major>"))
 final class DenseColumnMatrix(initDefault:Boolean = true) extends MatrixBase {
 
-  def this(nrow: Int, ncol: Int, ctx: Context = new Context()) {
+  def this(nrow: Int, ncol: Int) {
+    this(false)
+    allocate(nrow, ncol, new Context())
+  }
+
+  def this(nrow: Int, ncol: Int, ctx: Context) {
     this(false)
     allocate(nrow, ncol, ctx)
   }
 
-  def this(data: DoublePointer, nrow: Int, ncol: Int, ctx: Context = new Context(Context.MAIN_MEMORY)) {
```

Comment:
@andrewpalumbo can you try

`mvn test -Dscala.version=2.11.8 -Dscala.compat.version=2.11`

The reason I believe this works is- 
```
<dependency>
      <groupId>org.scalatest</groupId>
      <artifactId>scalatest_${scala.compat.version}</artifactId>
    </dependency>
```
where scala-compat-version is by default 2.10.  So when you `mvn test` you are doing so with scala-test_2.10, and that's where its losing its mind.

(Also explains why these were passing in Travis)